charts/dsp-atlantis/values.yaml

```diff
@@ -12,3 +13,9 @@ atlantis:
   fullnameOverride: atlantis
   serviceAccount:
     name: atlantis
+azure:
+  enabled: false
+  certificateFile: "azure-cert.pfx"
+vault:
+  azure:
+    pathPrefix: /path/to/azure/secrets
```

Comment:
This seems like a dummy value, will anything that cares attempt to use it?

Reply:
It is a dummy value it gets overriden in [this pr](https://github.com/broadinstitute/terra-helm-definitions/pull/53). 
That is the only place these values are used in our setup. 

When we auto generate documentation for this chart that dummy value will show up as in example which imo is better for documentation than leaving the real vault path in there.